commerce-theme-minium/commerce-theme-minium/src/css/minium/_mixins.scss

```diff
@@ -33,6 +28,11 @@
 	padding: 12px 30px;
```

Comment:
`@mixin card-title` should still be moved before `@mixin minium-typo-title`. Your choice whether `@mixin panel-title` should go with it or not